Taskfile.dist.yaml

```diff
@@ -16,15 +37,239 @@ tasks:
         done
       - git signatures push {{.CLI_ARGS}}
 
+  _build:cgo:dev: &build_cgo_dev
+    dir: "{{.dir}}"
+    cmds:
+      - go build {{if eq .raceDetectorEnabled "true" }}-race{{end}} -o "{{.outputDir | default (printf "./build/%s-%s" .targetOS .targetArch)}}/trdl{{if (eq .targetOS "windows")}}.exe{{end}}"
+        -compiler gc -ldflags="{{.cgoLDFlags}}" -tags="{{.cgoTags}}" {{.extraGoBuildArgs}} {{.CLI_ARGS}} {{.pkg | default .package}}
```

Comment:
Let's just have `{{ .package }}` instead of `{{.pkg | default .package}}`, and .package should always be specified explicitly. No defaults needed.